controllers/awsmachine_controller.go

```diff
@@ -836,8 +836,8 @@ func (r *AWSMachineReconciler) registerInstanceToV2LB(machineScope *scope.Machin
 	_, registered, err := elbsvc.IsInstanceRegisteredWithAPIServerLB(instance)
 	if err != nil {
 		r.Recorder.Eventf(machineScope.AWSMachine, corev1.EventTypeWarning, "FailedAttachControlPlaneELB",
-			"Failed to register control plane instance %q with load balancer: failed to determine registration status: %v", instance.ID, err)
-		return errors.Wrapf(err, "could not register control plane instance %q with load balancer - error determining registration status", instance.ID)
+			"Failed to register control plane instance %q with network load balancer: failed to determine registration status: %v", instance.ID, err)
```

Comment:
Same here, since there are a multitude of load balancer types, this is a generic message.

Reply:
Right, the v2 LB could be of different type. I made the wrong assumption here.

My point here was to clearly distinguish the log messages, since that would've led me into the right direction earlier, namely that the controller tried to attach to a v2 LB, not a classic LB. In general, it's much easier to have unique error messages. So can we keep at least the word "classic" inside `registerInstanceToClassicLB` since that one is specifically for that load balancer type!?